src/flow.py

```diff
@@ -1728,6 +1728,15 @@ def set_account_lock(self, username, lock_type, sid=0, timeout=None):
             timeout=timeout,
         )
 
+    def ping(self, sid=0, timeout=None):
+        """contact the server and report the result"""
+        sid = self._get_session_id(sid)
+        self._run(
+            method="Ping",
+            SessionID=sid,
+            timeout=timeout,
+        )
```

Comment:
As flowapp will return "null" if all is ok, then please return the result of self._run() (just for consistence)